lib/ChangeTypes.js

```diff
@@ -42,8 +42,10 @@ module.exports = {
   // e.g. x + 2 + x^2 + x + 4 -> x^2 + (x + x) + (4 + 2)
   COLLECT_LIKE_TERMS: 'COLLECT_LIKE_TERMS',
 
-  // MULTIPLYING CONSTANT POWERS
-  // e.g. 10^2 * 10^3 -> 10^(2+3)
+  // MULTIPLYING/DIVIDING CONSTANT POWERS
+
+  // e.g. 10^2 * 10^3 -> 10^(3+2)
+  // e.g. 10^4 / 10^2 -> 10^(4-2)
   COLLECT_CONSTANT_EXPONENTS: 'COLLECT_CONSTANT_EXPONENTS',
```

Comment:
can you have two changeGroups?

`COLLECT_CONSTANT_EXPONENTS_MULT` and `COLLECT_CONSTANT_EXPONENTS_DIV` for example